config.go

```diff
@@ -139,6 +139,7 @@ func (c *Config) ensureMembershipListeners() {
 	}
 }
 
+// AddFlakeIDGenerator validates the values and adds new FlakeIDGeneratorConfig with the given name.
```

Comment:
Not related to this line, but can you check if a flake ID generator was created with the same name?

Reply:
Should we assume it as an error or just override it like we do in both _client and member_ configs in Java (might be valid for other clients as well, not sure about them) - i.e. `Config#addMapConfig`, `Config#addListConfig`, `Config#addFlakeIdGeneratorConfig`, etc.? Asking since I feel this is the config idiom we follow so far.

Reply:
Let's make it an error. That's what we did everywhere else, e..g, addding serialization factories.

Reply:
addressed in fa6c1da